From 170857800f2a84b4cdb8fefe9a698efe5509837a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Fri, 16 Aug 2019 09:29:54 +0200
Subject: [PATCH] Rename coliding hash functions
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Ported from Fedora recode patch:

    commit 5cd1d43a39e4679ed265efcdf4bdc3da517e3de5 (tag: recode-3_6-16)
    Author: Bill Nottingham <notting@fedoraproject.org>
    Date:   Fri May 27 02:52:25 2005 +0000

        auto-import recode-3.6-16 on branch devel from recode-3.6-16.src.rpm

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 lib/hash.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/lib/hash.h b/lib/hash.h
index 562f5e6..65d967e 100644
--- a/lib/hash.h
+++ b/lib/hash.h
@@ -24,6 +24,11 @@
 #ifndef HASH_H_
 # define HASH_H_
 
+#define hash_lookup recode_hash_lookup
+#define hash_delete recode_hash_delete
+#define hash_free   recode_hash_free
+#define hash_insert recode_hash_insert
+
 # include <stdio.h>
 # include <stdbool.h>
 
-- 
2.21.0

